docs/samples/browser-plugin-meetings/app.js

```diff
@@ -978,30 +1000,173 @@ async function getUserMedia(constraints = {audio: true, video: true}) {
+      let effect;
+      effect = await localMedia.cameraTrack.getEffect("virtual-background");
+
+      if (!effect) {
+        effect = await webex.meetings.createVirtualBackgroundEffect();
+        await localMedia.cameraTrack.addEffect("virtual-background", effect);
+        meetingStreamsLocalVideo.srcObject = localMedia.cameraTrack.underlyingStream;
+        handleTrackUpdated(effect, localMedia.cameraTrack, meetingStreamsLocalVideo, 'local camera');
+      }
+
+      await effect.enable();
+
+      toggleVbgBtn.innerText = "Disable VBG"
+      console.log('MeetingControls#handleVbg() :: successfully applied virtual background to local camera track');
+    }
+    else {
+      console.log('MeetingControls#handleVbg() :: disabling virtual background from local camera track');
+
+      const effect = await localMedia.cameraTrack.getEffect("virtual-background");
+      await effect.disable();
+      toggleVbgBtn.innerText = "Enable VBG"
+      console.log('MeetingControls#handleVbg() :: successfully disabled virtual background from local camera track');
+    }
+  }
+  catch (e) {
+    console.log('MeetingControls#handleVbg() :: Error applying background effect!');
+    console.error();
+
+    return Promise.reject(e);
+  }
+}
+
+async function loadMicrophone(constraints) {
+  try {
+    console.log('MeetingControls#loadMicrophone() :: using webrtc-core local microphone track');
+    const audioConstraints = {...constraints};
+
+    console.log('MeetingControls#loadMicrophone() :: getting microphone track with constraints: ', audioConstraints);
     localMedia.microphoneTrack = await webex.meetings.mediaHelpers.createMicrophoneTrack(audioConstraints);
 
-    handleLocalTrackEvents(localMedia.microphoneTrack, meetingStreamsLocalAudio, 'local microphone');
+    handleTrackUnpublished(localMedia.microphoneTrack, meetingStreamsLocalAudio, 'local microphone');
 
     meetingStreamsLocalAudio.srcObject = localMedia.microphoneTrack.underlyingStream;
+    runPublishedStateTags()
+    console.log('MeetingControls#loadMicrophone() :: Successfully got microphone track:', localMedia.microphoneTrack);
+    return;
   }
+  catch (e) {
+    console.log('MeetingControls#loadMicrophone() :: Error getting microphone stream!');
+    console.error();
 
-  if (constraints.video) {
-    const videoConstraints = {...localMedia.videoConstraints[localVideoQuality[localResolutionInp.value]], ...constraints.video};
+    return Promise.reject(e);
+  }
+}
 
-    console.log('MeetingControls#getUserMedia() :: getting camera track with constraints: ', videoConstraints);
-    localMedia.cameraTrack = await webex.meetings.mediaHelpers.createCameraTrack(videoConstraints);
+async function handleBNR() {
+  try {
+    if (toggleBNRBtn.innerText === "Enable BNR") {
+      console.log('MeetingControls#handleBNR() :: applying BNR to local microhone track');
 
-    handleLocalTrackEvents(localMedia.cameraTrack, meetingStreamsLocalVideo, 'local camera');
+      let effect;
```

Comment:
We can declare this outside `if` and use the same inside else block as well

Reply:
done